### PR TITLE
PPS about draft

### PR DESCRIPTION
**Pull request by vv-monsalve**
@RosaWagner This PR adds a drafted Article to test how the contents would look.

### CODE REVIEW

**Comment by vv-monsalve** [comment]
The changes are still not visible. I'm testing the inclusion of a table with multiple small images (the emojis), and to do it, I've tried to add a "head" with some styles. I'm wondering if this is even possible, or if that is what is causing it to not work.

@nathan-williams could you please confirm about the above or if the "style" property should be added to each tag instead?

File: ofl/playpensans/article/ARTICLE.en_us.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    border-bottom: 1px dotted #d2d2d2;
+    padding: 0.3rem;
+    text-align: center;
+}
+td {
+    font-size: 40px;
+}
+</style>
+</head>
+<body>
```
Comment (RosaWagner) [inline ofl/playpensans/article/ARTICLE.en_us.html:15]: I think everything above that line should be removed, may come in conflict with the website.

File: ofl/playpensans/article/ARTICLE.en_us.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+</p>
+<p>    
+Playpen Sans has eight weights emojis for breezy and encouraging uses and supports over 150 Latin-based languages. Its straight and curved endings for ‘i, l, y’, the two-story ‘a’, and optional shapes for ‘f, G, I, M’ are clarifying features. The family was made for non-designers and to shine within short, informal settings: greeting cards and invitations, casual signs and documents, and of course, children’s books and educational materials, comic books, and graphic novels. It also comes in a variable font version for developers and designers who value ultimate control, data savings, and design superiority.
+</p>
+
+<p>    
+Playpen Sans is the extension of technological and aesthetic potential, showing the best of both worlds with its digital capabilities and casual, handmade look. Is it spontaneous? Is it authentic? Thankfully, yes, and yes.
+</p>
+<p>
+To contribute, see <a href="https://github.com/TypeTogether/Playpen-Sans.git" target="_blank">github.com/TypeTogether/Playpen-Sans</a>
```
Comment (RosaWagner) [inline ofl/playpensans/article/ARTICLE.en_us.html:38]: also target_blank should be removed

File: ofl/playpensans/article/ARTICLE.en_us.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    </tr>
+    <tr>
+        <td>🎨</td>
+        <td><img src="e-color-palette.png"></td>
+    </tr>
+    <tr>
+        <td style="font-size: 30px;">👆</td>
+        <td><img src="e-index-pointing.png"></td>
+    </tr>
+</table>
```
Comment (RosaWagner) [inline ofl/playpensans/article/ARTICLE.en_us.html:63]: the `table`, `tr` and `td` elements are not listed in the allowed elements for the article

File: ofl/playpensans/article/ARTICLE.en_us.html (hunk 1 above)
Comment (vv-monsalve) [reply ofl/playpensans/article/ARTICLE.en_us.html:15]: Yeah, was hesitant, but was worth trying it.

File: ofl/playpensans/article/ARTICLE.en_us.html (hunk 3 above)
Comment (vv-monsalve) [reply ofl/playpensans/article/ARTICLE.en_us.html:63]: AFAIR Chris confirmed with Nate that tables were also supported.

**Pull request merged**